core/src/main/java/org/jdbi/v3/core/argument/JavaTimeZoneIdArgumentFactory.java

```diff
@@ -26,6 +28,6 @@ public JavaTimeZoneIdArgumentFactory() {
 
     @Override
     protected Argument build(ZoneId value, ConfigRegistry config) {
-        return new LoggableToStringOrNPEArgument<>(value);
+        return new LoggableBinderArgument<>(Objects.toString(value, null), PreparedStatement::setString);
```

Comment:
in this case, it's almost certainly a bug -- binding a `null` `ZoneId` should be valid and simply bind null.

Reply:
v4 Argument should probably add a `getValue()` method or something, so we can stop doing these hacks

Reply:
Or possibly, we could refactor bindings so that the value and the Argument are stored in some other type